README.rst

```diff
@@ -0,0 +1,218 @@
+Mail BB Gun
+===========
+
+A baby version of MailGun
+
+Requirements
+------------
+
+* Python 3.6
+* Docker
+* SMTP Server account
+
+Setup
+-----
+
+Start docker services.
+
+.. code-block:: bash
+
+    docker-compose up -d
+
+Install Python dependencies.  Using a Virtual Environment is highly recommended.
+
+.. code-block:: bash
+
+   pip install -r requirements.txt
+
+Generate the database password.  Be sure to save it in a safe place.
+
+.. code-block:: bash
+
+    export PG_PASSWORD=$(dd if=/dev/urandom bs=1 count=24 2>/dev/null | hexdump -v -e '/1 "%02X"')
+    echo $PG_PASSWORD
+
+Export your SMTP password.
+
+.. code-block:: bash
+
+    export SMTP_PASSWORD=YOUR_SMTP_PASSWORD
+
+Copy the sample config file and update it with relevant SMTP account values for your deployment.
+
+.. code-block:: bash
+
+    cp config.py.sample config.py
+
+Add the mailbbgun user to postgresql.
```

Comment:
What replication strategies does postgres have? What if the postgres goes down? How this could be handled from the postgres side and from the service side?